src/hooks/use-account.ts

```diff
@@ -31,7 +31,9 @@ export const useAccount = () => {
       return account.owner?.id === viewer.id
     })
 
-  const isGiftMembership = account?.subscriptions?.[0]?.type === 'gift'
+  const isGiftMembership =
+    account?.subscriptions?.[0]?.type === 'gift' &&
+    account?.subscriptions?.[0]?.status === 'active'
```

Comment:
made `isGiftMembership` dependent on `status` here

Reply:
It should check for `past_due` and `trialing` as valid active options too. (Although, we shouldn't have any accounts in the trialing state.)

Alternatively, we could just have `egghead-rails` return `enabled` or `disabled` so we're not duplicating the logic? See statuses from `egghead-rails`:
https://github.com/skillrecordings/egghead-rails/blob/main/app/models/account_subscription.rb#L2